Mathlib/Algebra/Order/Hom/Monoid.lean

```diff
@@ -266,8 +266,6 @@ theorem monotone_iff_map_nonpos : Monotone (f : α → β) ↔ ∀ a ≤ 0, f a
 theorem antitone_iff_map_nonneg : Antitone (f : α → β) ↔ ∀ a ≤ 0, 0 ≤ f a :=
   monotone_comp_ofDual_iff.symm.trans <| monotone_iff_map_nonneg (α := αᵒᵈ) (iamhc := iamhc) _
 
-variable [AddLeftStrictMono β]
```

Comment:
Just to point out that there is a strict improvement here: this typeclass was not necessary before, but this wasn't known to typeclass inference at this stage of the library.